{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-4363",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/178960",
                    "name": "ibm-db2-cve20204363-bo (178960)",
                    "refsource": "XF",
                    "tags": []
                },
                {
                    "url": "https://www.ibm.com/support/pages/node/6242332",
                    "name": "https://www.ibm.com/support/pages/node/6242332",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "IBM DB2 for Linux, UNIX and Windows (includes DB2 Connect Server) 9.7, 10.1, 10.5, 11.1, and 11.5 is vulnerable to a buffer overflow, caused by improper bounds checking which could allow a local attacker to execute arbitrary code on the system with root privileges. IBM X-Force ID: 178960."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-07-01T15:15Z",
    "lastModifiedDate": "2020-07-01T15:15Z"
}